gatsby-browser.js

```diff
@@ -1,7 +1,15 @@
+import * as React from "react"
+import {StoreProvider} from "./src/components/ga4/EnhancedEcommerce/store-context"
 import CustomLayout from "./gatsby/wrapRootElement.js"
+import "./src/styles/ecommerce/variables.css"
```

Comment:
Not strictly necessary, but it'd probably be better if styling was consistent accross this repo. Currently, the main form of styling is through react mui's `useStyles` hook ([example](https://github.com/googleanalytics/ga-dev-tools/blob/main/src/components/Layout/useStyles.ts#L9))

Reply:
Thanks, didn't realize that. Looks like I need to move the css variables to Layout/useStyles.ts ? I'll probably do this in a separate PR then.

Reply:
Following up in separate PR sgtm. 

There are a lot of styling options for mui. I usually recommend doing styles at the `./src/components/index.tsx` level. You can define the hook inline, or pull it out to its own file. Kinda just depends on preference. One of the big things to keep in mind with mui styling is you don't have to worry about global namespcing because it generates unique class names based on your definitions. This makes it easier to have local css that is unique to the component it's working on.